{"name": "test_div[10-0-除数不能为0]", "status": "broken", "statusDetails": {"message": "ZeroDivisionError: division by zero", "trace": "self = <test_case.test_case.TestCase object at 0x000001BFA4B88E00>, a = 10, b = 0, expected = '除数不能为0'\n\n    @pytest.mark.parametrize('a,b,expected', GetData.get_div_data())\n    def test_div(self, a, b, expected):\n        caculator = Caculator(a, b)\n>       assert caculator.div() == expected\n\ntest_case.py:47: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <func.func.Caculator object at 0x000001BFA4BB1F40>\n\n    def div(self):\n        if self.a > 99 or self.a < -99 or self.b > 99 or self.b < -99:\n            print(\"请输入范围为【-99, 99】的整数或浮点数\")\n            return \"参数大小超出范围\"\n    \n>       return self.a / self.b\nE       ZeroDivisionError: division by zero\n\n..\\func\\func.py:43: ZeroDivisionError"}, "parameters": [{"name": "a", "value": "10"}, {"name": "b", "value": "0"}, {"name": "expected", "value": "'除数不能为0'"}], "start": 1713491156899, "stop": 1713491156899, "uuid": "f022a7cb-2222-4285-a4a2-2c337fb75f9a", "historyId": "5ae287271cfa546515b783a0946d1457", "testCaseId": "b12eca9b108b11739c311ed63df77077", "fullName": "test_case.test_case.TestCase#test_div", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "wangmin"}, {"name": "thread", "value": "10380-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case"}]}